core/vec3.h

```diff
@@ -2,23 +2,32 @@
 #define VEC3_H
 
 #include "constants.h"
+
+#include <ctime>
+#include <exception>
 #include <initializer_list>
 #include <iostream>
+#include <random>
+#include <sstream>
 
 namespace core {
 
 class Vec3 {
+public:
+  static const Vec3 kVecZero; // Vec3(0, 0, 0)
+  static const Vec3 kVecX;    // Vec3(1, 0, 0)
+  static const Vec3 kVecY;    // Vec3(0, 1, 0)
+  static const Vec3 kVecZ;    // Vec3(0, 0, 1)
+
+  // creates Vec3 object, which has every coordinate drawn from Normal
+  // Distribution.
+  static Vec3 getRandomVec();
```

Comment:
That is super bad. We talked about this no?

This depends on some static state, that you will never be able to change in test or control easily in a main program. Remove.

I mentioned before to completely remove this randomness in tests, as it does not bring much value, for such a huge increase in complexity, and loss of readability.

Besides, it is unused.